Sources/_StringProcessing/ByteCodeGen.swift

```diff
@@ -110,7 +111,8 @@ fileprivate extension Compiler.ByteCodeGen {
   mutating func emitQuotedLiteral(_ s: String) {
```

Comment:
@milseman Curious to get your thoughts on combining the `emitReverseQuotedLiteral` and `emitQuotedLiteral` like I've done here. I'm a bit torn on whether it's better to duplicate code or add complexity here.

Reply:
Obviously this doesn't preclude the need for unit tests but I think it'll make maintenance easier in the long term if there aren't two nearly identical copies of functions all over the place

Reply:
Avoiding duplication in the compilation makes sense. For the processor itself, I don't want a dynamic branch at the lowest-most part of matching to choose between forwards and reverse, so there's more nuance there. We might be able to extract functionality into common shared code, or we might do some kind of static meta programming.